.github/workflows/linux-eic-shell.yml

```
@@ -514,27 +511,110 @@ jobs:
           doxygen Doxyfile
           cp -r docs publishing_docs
           mv html publishing_docs/doxygen
+      - uses: actions/download-artifact@v3
+        with:
+          name: jana.dot
+          path: publishing_docs/dot/
       - uses: actions/upload-artifact@v3
         with:
           name: docs
           path: publishing_docs/
           if-no-files-found: error
+
+  list-open-prs:
+    runs-on: ubuntu-latest
+    outputs:
+      json: ${{ steps.remap.outputs.json }}
+    steps:
+      - uses: octokit/request-action@v2.x
+        id: query
+        with:
+          route: GET /repos/${{ github.repository }}/pulls?state=open
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Remap open PRs
+        id: remap
+        uses: nickofthyme/object-remap@v2
+        with:
+          include.*.pr: ${{ toJSON(fromJSON(steps.query.outputs.data).*.number) }}
+
+  get-docs-from-open-prs:
+    runs-on: ubuntu-latest
+    needs:
+      - build-docs
+      - list-open-prs
+    strategy:
+      matrix: ${{ fromJson(needs.list-open-prs.outputs.json) }}
```

```suggestion
      matrix: ${{ fromJSON(needs.list-open-prs.outputs.json) }}
```
This is official spelling https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson